config/webpack.mobile.target.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+
+const production = /:production$/.test(process.env.NODE_ENV)
+
+module.exports = {
+  entry: path.resolve(__dirname, '../mobile/src/main'),
+  output: {
+    path: path.resolve(__dirname, '../mobile/www')
+  },
+  module: {
+    noParse: [
+      /localforage\/dist/
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      __ALLOW_HTTP__: !production,
+      __TARGET__: JSON.stringify('mobile')
```

Comment:
What's the aim? As `JSON.stringify('mobile') === '"mobile"'` :neckbeard: 

In `src/` we would be able to write conditions like:
```javascript
if (__TARGET__ === 'mobile') {
  // do something awesome for mobile application
}
```